packages/frontend/lib/index.ts

```diff
@@ -82,11 +82,24 @@ export default class Nango {
     /**
      * Creates a new unauthenticated connection using the specified provider configuration key and connection ID
      * @param providerConfigKey - The key identifying the provider configuration on Nango
-     * @param connectionId -  The ID of the connection
+     * @param connectionId -  Optional. The ID of the connection
      * @param connectionConfig - Optional. Additional configuration for the connection
      * @returns A promise that resolves with the authentication result
      */
-    public async create(providerConfigKey: string, connectionId: string, connectionConfig?: ConnectionConfig): Promise<AuthResult> {
+    public async create(providerConfigKey: string, connectionConfig?: ConnectionConfig): Promise<AuthResult>;
+    public async create(providerConfigKey: string, connectionId: string, connectionConfig?: ConnectionConfig): Promise<AuthResult>;
+    public async create(
+        providerConfigKey: string,
+        connectionIdOrConnectionConfig?: string | ConnectionConfig,
+        moreConnectionConfig?: ConnectionConfig
+    ): Promise<AuthResult> {
```

Comment:
method overload to allow passing `connectionConfig` without having to specify an `undefined` connectionId